### PR TITLE
ibm5170 - New working software list additions

### PR DESCRIPTION
**Pull request by ArcadeShadow**
Added: [tetriscw] Tetris Classic (Windows 3.x release)

### CODE REVIEW

File: hash/ibm5170.xml [hunk 1]
```diff
@@ -15481,6 +15481,24 @@ license:CC0
...
+		<info name="developer" value="Sphere" />
+		<info name="version" value="Version 1.0 10/23/92" />
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="1474560">
+				<rom name="Tetris Classic (windows 3.x release) [Spectrum Holobyte] [1992] [3.5DD] [Disk 2 of 2].img" size="1474560" crc="79804dc4" sha1="2ce4e7641a5d9422d79a85d300a7c0f9140ade88"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="Tetris Classic (windows 3.x release) [Spectrum Holobyte] [1992] [3.5HD] [Disk 1 of 2].img" size="737280" crc="655bda8e" sha1="2a2053439e2bec82cb0be8cd4bae8035e9cf33a7"/>
```
Comment (DopefishJustin) [inline hash/ibm5170.xml:15497]: Disk 1 and 2 are the wrong way around?

Comment (ArcadeShadow) [reply hash/ibm5170.xml:15497]: Sorry, yes they are, my mistake. Can you switch them? please.

Comment (happppp) [reply hash/ibm5170.xml]: Ok it's done. It is something you could have done too by simply doing an additional commit to the PR.

**Comment by ArcadeShadow** [comment]
I correct my entry, and I think now it's OK. Thanks!

**Comment by happppp** [comment]
Ok great

**Pull request merged**